Makefile

```diff
@@ -1,7 +1,13 @@
+EMPTY_STR=""
+
 .PHONY: prepare-release
 prepare-release:
 	git submodule update --recursive --remote --init
+ifeq ($(AUTONOMY_VERSION), )
 	cd open-autonomy/; git fetch origin --tags --force; git checkout $$(git tag --sort=committerdate | tail -n 1)
+else
+	cd open-autonomy/; git fetch origin --tags --force; git checkout $$AUTONOMY_VERSION
+endif
```

Comment:
@jmoreira-valory If you want to bump open-autonomy to a specific version tag use

```
AUTONOMY_VERSION=version make prepare-release
```